### PR TITLE
lp-to-git-users: adding asciiprod

### PR DESCRIPTION
**Pull request by asciiprod**
Mapped from lp-markusschade

### CODE REVIEW

**Review by OddBloke — APPROVED** [review]
Confirmed via https://code.launchpad.net/~lp-markusschade/cloud-init/+git/cloud-init/+merge/392777

**Comment by OddBloke** [comment]
@asciiprod I'm not able to update this branch in order to land it because you've disabled "Allow edits from maintainer": as we require branches to be up-to-date with master before they can merge them (and this is enforced via GH UI), we'll need you to have this up-to-date before we can land it.

(If you can't change it, I totally understand, but it'll take less synchronisation to land your changes if you _can_ enable that option going forward. :+1:)

**Comment by igalic** [comment]
BTW, if you add your commit email to your github account, your patches will also be properly associated with it.

**Pull request merged**
